### PR TITLE
Add descriptions & placeholders to blog posts

### PR DESCRIPTION
**Pull request by yasik78**
Featured Blog Posts paragraph descriptions & placeholders #484
## Steps for review

- [x] login as admin
- [x] Go to /node/add/landing_page
- [x] Add paragraf block Simple content 
- [x] Check description for fields and compare with docs.  https://github.com/ymcatwincities/openy/blob/8.x-1.x/docs/Content%20structure/Paragraphs/Featured%20Blog%20Posts.md

### CODE REVIEW

**Comment by gundevel** [comment]
Can one of the admins verify this patch? Use "o+k to test" or ''t+est this please" for manual build execution.

**Comment by Sanchiz** [comment]
ok to test

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============

--------------
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1530/

**Build Log**
*last 1000 lines*

 ```
GitHub pull request #556 of commit 6b42b308ee698b3ff2b2e690269786962f992732 automatically merged.
GitHub pull request #556 of commit 6b42b308ee698b3ff2b2e690269786962f992732 automatically merged.
GitHub pull request #556 of commit 6b42b308ee698b3ff2b2e690269786962f992732 automatically merged.
Building in workspace /var/lib/jenkins/jobs/PR_BUILDER/workspace@8
Cloning the remote Git repository
Using shallow clone
Cloning repository https://github.com/ymcatwincities/openy.git
 > git init /var/lib/jenkins/jobs/PR_BUILDER/workspace@8/sourcecode # timeout=10
Fetching upstream changes from https://github.com/ymcatwincities/openy.git
 > git --version # timeout=10
using .gitcredentials to set credentials
 > git config --local credential.helper store --file=/tmp/git3102259783447826979.credentials # timeout=10
 > git -c core.askpass=true fetch --tags --progress https://github.com/ymcatwincities/openy.git +refs/heads/*:refs/remotes/origin/* --depth=1 # timeout=5
 > git config --local --remove-section credential # timeout=10
 > git config remote.origin.url https://github.com/ymcatwincities/openy.git # timeout=10
 > git config --add remote.origin.fetch +refs/heads/*:refs/remotes/origin/* # timeout=10
 > git config remote.origin.url https://github.com/ymcatwincities/openy.git # timeout=10
Cleaning workspace
 > git rev-parse --verify HEAD # timeout=10
No valid HEAD. Skipping the resetting
 > git clean -fdx # timeout=10
Fetching upstream changes from https://github.com/ymcatwincities/openy.git
using .gitcredentials to set credentials
 > git config --local credential.helper store --file=/tmp/git1528914065119486361.credentials # timeout=10
 > git -c core.askpass=true fetch --tags --progress https://github.com/ymcatwincities/openy.git +refs/pull/*:refs/remotes/origin/pr/* # timeout=5
 > git config --local --remove-section credential # timeout=10
 > git rev-parse origin/pr/556/merge^{commit} # timeout=10
 > git branch -a --contains 30edbcb97598016c876adadbebec1ef54d3bade4 # timeout=10
 > git rev-parse remotes/origin/pr/556/merge^{commit} # timeout=10
Checking out Revision 30edbcb97598016c876adadbebec1ef54d3bade4 (origin/pr/556/merge)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f 30edbcb97598016c876adadbebec1ef54d3bade4
First time build. Skipping changelog.
[workspace@8] $ /bin/sh -xe /tmp/hudson7978082977106629130.sh
+ rm -Rf /var/lib/jenkins/jobs/PR_BUILDER/workspace@8/vm
+ cp -R ../vm /var/lib/jenkins/jobs/PR_BUILDER/workspace@8/vm
+ cd /var/lib/jenkins/jobs/PR_BUILDER/workspace@8/vm
+ git pull
Already up-to-date.
[workspace@8] $ ansible-playbook /var/lib/jenkins/jobs/PR_BUILDER/workspace@8/vm/cibox/jobs/deploy_code.yml -i /tmp/inventory3094032667715714484.ini -f 5 -e workspace=/var/lib/jenkins/jobs/PR_BUILDER/workspace@8 -e build_number=1530 -e server_docroot_folder=/var/www -e deploy_type=local -e profile_source=/var/lib/jenkins/jobs/PR_BUILDER/workspace@8/sourcecode -e source=/var/lib/jenkins/jobs/PR_BUILDER/workspace@8/vm/docroot -e profile_destination=profiles/contrib/openy -e build_folder_prefix=build -i localhost, --connection=local

PLAY [localhost] ************************************************************** 

GATHERING FACTS *************************************************************** 
ok: [localhost]

TASK: [../custom/cibox-code-deploy | Generate commentinfo.md file] ************ 
changed: [localhost]

TASK: [../custom/cibox-code-deploy | Make sure folder is not exist] *********** 
ok: [localhost]

TASK: [../custom/cibox-code-deploy | Sync files into docroot] ***************** 
changed: [localhost]

TASK: [../custom/cibox-code-deploy | Sync OpenY profile] ********************** 
changed: [localhost]

TASK: [../custom/cibox-code-deploy | Folder permissions] ********************** 
changed: [localhost]

PLAY RECAP ******************************************************************** 
localhost                  : ok=6    changed=4    unreachable=0    failed=0   

[workspace@8] $ ansible-playbook /var/lib/jenkins/jobs/PR_BUILDER/workspace@8/vm/cibox/jobs/build.yml -i /tmp/inventory6535460025958737028.ini -f 5 -e workspace=/var/lib/jenkins/jobs/PR_BUILDER/workspace@8 -e build_number=1530 -e server_docroot_folder=/var/www -e build_folder_prefix=build -i localhost, --connection=local

PLAY [localhost] ************************************************************** 

GATHERING FACTS *************************************************************** 
ok: [localhost]

TASK: [Folder permissions] **************************************************** 
changed: [localhost]

PLAY RECAP ******************************************************************** 
localhost                  : ok=2    changed=1    unreachable=0    failed=0   

[workspace@8] $ ansible-playbook /var/www/build1530/reinstall.yml -f 5 -e php_env_vars=APP_ENV=dev -e mysql_user=root -e mysql_password=root -e mysql_db=drupal1530 -e drupal_folder=/var/www/build1530 -e site_url=http://openy-dev.ffwua.com/build1530 -e pp_environment=default -e run_reinstall=true -i localhost, --connection=local

PLAY [localhost] ************************************************************** 

GATHERING FACTS *************************************************************** 
ok: [localhost]

TASK: [Include specific variables per environment] **************************** 
ok: [localhost]

TASK: [Getting timestamp for backup] ****************************************** 
skipping: [localhost]

TASK: [Getting name for backup] *********************************************** 
skipping: [localhost]

TASK: [Creating backup folder for environment] ******************************** 
skipping: [localhost]

TASK: [Making regular backup from {{ local_backup_environment }} environment] *** 
skipping: [localhost]

TASK: [Make new backup as latest] ********************************************* 
skipping: [localhost]

TASK: [Set writable cache dir] ************************************************ 
ok: [localhost]

TASK: [Set writable tmp dir] ************************************************** 
changed: [localhost]

TASK: [Set writable backup dir] *********************************************** 
ok: [localhost]

TASK: [Set writable default dir] ********************************************** 
changed: [localhost]

TASK: [Set writable files dir] ************************************************ 
changed: [localhost]

TASK: [Set writable private dir] ********************************************** 
changed: [localhost]

TASK: [Start the MySQL service] *********************************************** 
ok: [localhost]

TASK: [Getting name for settings (WINDOWS)] *********************************** 
skipping: [localhost]

TASK: [Getting name for settings (UNIX)] ************************************** 
skipping: [localhost]

TASK: [Remove settings.php] *************************************************** 
skipping: [localhost]

TASK: [Creating settings.php] ************************************************* 
skipping: [localhost]

TASK: [Adding database variable to settings.php] ****************************** 
skipping: [localhost]

TASK: [Add general pre-settings to settings.php] ****************************** 
skipping: [localhost]

TASK: [Add environment specific pre-settings] ********************************* 
skipping: [localhost] => (item={'status': True, 'name': '$settings["hash_salt"]', 'value': 'g1W4KZgWCeWh7Af9_gSb9D9HOiNaBdWPvQD-8x0yWpcgNpTBppaHR5VNGQBGvh2k8P8uWa1V8w'})
skipping: [localhost] => (item={'status': False, 'name': '$config_directories["staging"]', 'value': 'sites/default/config/staging'})
skipping: [localhost] => (item={'status': True, 'name': '$config_directories["sync"]', 'value': 'sites/default/config/sync'})
skipping: [localhost] => (item={'status': True, 'name': '$settings["install_profile"]', 'value': u'openy'})
skipping: [localhost] => (item={'status': True, 'name': '$settings["file_public_path"]', 'value': u'sites/default/files'})
skipping: [localhost] => (item={'status': True, 'name': '$config["system.file"]["path.temporary"]', 'value': '/tmp'})

TASK: [Copy settings back (WINDOWS)] ****************************************** 
skipping: [localhost]

TASK: [Droping drupal database] *********************************************** 
skipping: [localhost]

TASK: [Creating drupal database] ********************************************** 
skipping: [localhost]

TASK: [Downloading latest backup to local environment] ************************ 
skipping: [localhost]

TASK: [Removing possible abandoned dump] ************************************** 
skipping: [localhost]

TASK: [Importing from remote backup] ****************************************** 
skipping: [localhost]

TASK: [Copy latest backup to build folder] ************************************ 
skipping: [localhost]

TASK: [Importing from local backup] ******************************************* 
skipping: [localhost]

TASK: [Remove local backup from build folder] ********************************* 
skipping: [localhost]

TASK: [Build project from drupal-org.make] ************************************ 
skipping: [localhost]

TASK: [Clear drush cache] ***************************************************** 
skipping: [localhost]

TASK: [Rebuilding drupal registry] ******************************************** 
skipping: [localhost]

TASK: [Clear Drupal cache] **************************************************** 
skipping: [localhost]

TASK: [Updating database] ***************************************************** 
skipping: [localhost]

TASK: [Apply pending entity schema updates] *********************************** 
skipping: [localhost]

TASK: [Download stage file proxy] ********************************************* 
skipping: [localhost]

TASK: [Enable stage file proxy] *********************************************** 
skipping: [localhost]

TASK: [Set configuration for stage file proxy] ******************************** 
skipping: [localhost]

TASK: [Set folder for stage file proxy] *************************************** 
skipping: [localhost]

TASK: [Updates listing] ******************************************************* 
skipping: [localhost]

TASK: [Droping drupal database] *********************************************** 
ok: [localhost]

TASK: [Creating drupal database] ********************************************** 
changed: [localhost]

TASK: [Remove settings.php] *************************************************** 
ok: [localhost]

TASK: [Build project from drupal-org.make] ************************************ 
changed: [localhost]

TASK: [Installing drupal] ***************************************************** 
failed: [localhost] => {"changed": true, "cmd": "APP_ENV=dev drush -y si openy --db-url=mysql://root:root@127.0.0.1:/drupal1530 --account-name=admin --account-pass=openy --uri=http://openy-dev.ffwua.com/build1530", "delta": "0:21:37.423059", "end": "2017-06-14 08:23:40.218050", "rc": 1, "start": "2017-06-14 08:02:02.794991", "warnings": []}
stderr: Starting Drupal installation. This takes a while. Consider using the [ok]
--notify global option.
Drupal\Core\Config\UnsupportedDataTypeConfigException: Invalid data  [error]
type in config
field.field.paragraph.featured_blogs.field_fblog_posts, found in
fileprofiles/contrib/openy/modules/openy_features/openy_prgf/modules/openy_prgf_featured_blogs/config/install/field.field.paragraph.featured_blogs.field_fblog_posts.yml
: yaml_parse(): reading error encountered during parsing: invalid
leading UTF-8 octet (line 1, column 1) in
/var/www/build1530/core/lib/Drupal/Core/Config/FileStorage.php:117
Stack trace:
#0
/var/www/build1530/core/lib/Drupal/Core/Config/FileStorage.php(130):
Drupal\Core\Config\FileStorage->read('field.field.par...')
#1
/var/www/build1530/core/lib/Drupal/Core/Config/ConfigInstaller.php(254):
Drupal\Core\Config\FileStorage->readMultiple(Array)
#2
/var/www/build1530/core/lib/Drupal/Core/Config/ConfigInstaller.php(504):
Drupal\Core\Config\ConfigInstaller->getConfigToCreate(Object(Drupal\Core\Config\FileStorage),
'', '', Array)
#3
/var/www/build1530/core/lib/Drupal/Core/Config/ConfigInstaller.php(467):
Drupal\Core\Config\ConfigInstaller->findDefaultConfigWithUnmetDependencies(Object(Drupal\Core\Config\FileStorage),
Array, Array)
#4
/var/www/build1530/core/lib/Drupal/Core/Extension/ModuleInstaller.php(141):
Drupal\Core\Config\ConfigInstaller->checkConfigurationToInstall('module',
'openy_prgf_feat...')
#5
/var/www/build1530/core/lib/Drupal/Core/ProxyClass/Extension/ModuleInstaller.php(83):
Drupal\Core\Extension\ModuleInstaller->install(Array, false)
#6 /var/www/build1530/core/includes/install.core.inc(1800):
Drupal\Core\ProxyClass\Extension\ModuleInstaller->install(Array,
false)
#7 /var/www/build1530/core/includes/batch.inc(252):
_install_module_batch('openy_prgf_feat...', 'OpenY Paragraph...',
Array)
#8 /var/www/build1530/core/includes/form.inc(874): _batch_process()
#9 /var/www/build1530/core/includes/install.core.inc(619):
batch_process(Object(Drupal\Core\Url), Object(Drupal\Core\Url))
#10 /var/www/build1530/core/includes/install.core.inc(540):
install_run_task(Array, Array)
#11 /var/www/build1530/core/includes/install.core.inc(117):
install_run_tasks(Array)
#12 /usr/local/src/drush/includes/drush.inc(726):
install_drupal(Object(Composer\Autoload\ClassLoader), Array)
#13 /usr/local/src/drush/includes/drush.inc(711):
drush_call_user_func_array('install_drupal', Array)
#14 /usr/local/src/drush/commands/core/drupal/site_install.inc(80):
drush_op('install_drupal', Object(Composer\Autoload\ClassLoader),
Array)
#15 /usr/local/src/drush/commands/core/site_install.drush.inc(249):
drush_core_site_install_version('openy', Array)
#16 /usr/local/src/drush/includes/command.inc(422):
drush_core_site_install('openy')
#17 /usr/local/src/drush/includes/command.inc(231):
_drush_invoke_hooks(Array, Array)
#18 /usr/local/src/drush/includes/command.inc(199):
drush_command('openy')
#19 /usr/local/src/drush/lib/Drush/Boot/BaseBoot.php(67):
drush_dispatch(Array)
#20 /usr/local/src/drush/includes/preflight.inc(66):
Drush\Boot\BaseBoot->bootstrap_and_dispatch()
#21 /usr/local/src/drush/drush.php(12): drush_main()
#22 {main}
stdout: You are about to create a /var/www/build1530/sites/default/settings.php file and DROP all tables in your 'drupal1530' database. Do you want to continue? (y/n): y

FATAL: all hosts have already failed -- aborting

PLAY RECAP ******************************************************************** 
           to retry, use: --limit @/var/lib/jenkins/reinstall.retry

localhost                  : ok=14   changed=6    unreachable=0    failed=1   

FATAL: command execution failed
hudson.AbortException: Ansible playbook execution failed
	at org.jenkinsci.plugins.ansible.AnsiblePlaybookBuilder.perform(AnsiblePlaybookBuilder.java:227)
	at org.jenkinsci.plugins.ansible.AnsiblePlaybookBuilder.perform(AnsiblePlaybookBuilder.java:200)
	at hudson.tasks.BuildStepCompatibilityLayer.perform(BuildStepCompatibilityLayer.java:75)
	at hudson.tasks.BuildStepMonitor$1.perform(BuildStepMonitor.java:20)
	at hudson.model.AbstractBuild$AbstractBuildExecution.perform(AbstractBuild.java:779)
	at hudson.model.Build$BuildExecution.build(Build.java:205)
	at hudson.model.Build$BuildExecution.doRun(Build.java:162)
	at hudson.model.AbstractBuild$AbstractBuildExecution.run(AbstractBuild.java:537)
	at hudson.model.Run.execute(Run.java:1741)
	at hudson.model.FreeStyleBuild.run(FreeStyleBuild.java:43)
	at hudson.model.ResourceController.execute(ResourceController.java:98)
	at hudson.model.Executor.run(Executor.java:381)
ERROR: Ansible playbook execution failed
```

File: modules/openy_features/openy_prgf/modules/openy_prgf_featured_blogs/config/install/field.field.paragraph.featured_blogs.field_fblog_posts.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ field_name: field_fblog_posts
 entity_type: paragraph
 bundle: featured_blogs
 label: 'Blog Posts'
-description: ''
+description: 'Multiple values. Reference field to Blog posts.�'
```
Comment (alexschedrov) [inline modules/openy_features/openy_prgf/modules/openy_prgf_featured_blogs/config/install/field.field.paragraph.featured_blogs.field_fblog_posts.yml]: Weird symbol at the end.
image

**Review by alexschedrov — CHANGES_REQUESTED** [review]

**Comment by bjwilkins** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1540 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1540 |
| Installation process | http://install.openy-dev.ffwua.com/build1540/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1540/

**Comment by bjwilkins** [comment]
Behat test results file URL: http://ci.openymca.org/build1540/profiles/contrib/openy/build/reports/behat

**Review by alexschedrov — APPROVED** [review]

**Pull request merged**